src/Board.py

```diff
@@ -148,33 +154,37 @@ def undoLastMove(self) -> None:
         self.currentSide = not self.currentSide
 
     def isCheckmate(self) -> bool:
-        #Game continue even after checkmate
+        # Game continue even after checkmate
         if len(self.getAllMovesLegal(self.currentSide)) == 0:
             for move in self.getAllMovesUnfiltered(not self.currentSide):
                 pieceToTake = move.pieceToCapture
-                if pieceToTake and pieceToTake.stringRep == "K":
+                if pieceToTake and pieceToTake.stringRep == 'K':
                     return True
         return False
 
     def isStalemate(self) -> bool:
-        return len(self.getAllMovesLegal(self.currentSide)) == 0 and not self.isCheckmate()
+        return (
+                len(self.getAllMovesLegal(self.currentSide)) == 0
+                and not self.isCheckmate()
```

Comment:
This only needs to be indented one times, not two times.

Reply:
like this? 
```
        return len(self.getAllMovesLegal(self.currentSide)) == 0 \
               and not self.isCheckmate()
```